### PR TITLE
Include vcomp140.dll in windows wheels

### PR DESCRIPTION
**Pull request by ogrisel**
This is a tentative fix for:

https://github.com/scikit-learn/scikit-learn/issues/15899

### CODE REVIEW

File: appveyor/vendor_vcomp140.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    """.format(dll_filename)))
+    return op.abspath(distributor_init)
+
+
+def main():
+    # TODO: use threadpoolctl to dynamically locate the right vcomp dll
+    # instead? This would require first in-place building scikit-learn
+    # to make it "importable".
+    if not os.path.exists(VCOMP140_SRC_PATH):
```
Comment (jeremiedbb) [inline appveyor/vendor_vcomp140.py]:
```suggestion
    if not op.exists(VCOMP140_SRC_PATH):
```

File: appveyor/vendor_vcomp140.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        raise ValueError("Could not find %r" % VCOMP140_SRC_PATH)
+
+    if not os.path.isdir("build"):
```
Comment (jeremiedbb) [inline appveyor/vendor_vcomp140.py]:
```suggestion
    if not op.isdir("build"):
```

**Comment by ogrisel** [comment]
Comments addressed. Thanks for the review.

**Comment by ogrisel** [comment]
I have not yet back-ported scikit-learn/scikit-learn#15929 to the 0.22.X branch yet, so this PR is targeting the master branch for now.

I will update again once the 0.22.1 PR is ready (there are still issues to fix on the milestone).

**Comment by ogrisel** [comment]
> I tested this on a new Windows 10 VM with python 3.8, where the pypi wheel failed and the artifact generated by this PR works.

Thanks for the additional manual confirmation :)

**Comment by ogrisel** [comment]
The random appveyor failure had also been observed elsewhere in the past and is being fixed here: https://github.com/scikit-learn/scikit-learn/pull/16003

**Pull request merged**